### DISCUSSION TITLE
How can I fix the nozzle temp?

### DISCUSSION THREAD

**Discussion by Giterdun2** [xblax/flashforge_ad5m_klipper_mod#115] [Q&A] [answered]
I have KlipperMod beta v00.04 installed on my 5M Pro.. When I send a file to the printer from Orca, the bed heats up to the profile temp that's set to 60, but the Nozzle only goes to 200, instead of 220 that is in the Orca profile. I have to manually raise it to 220 in Mainsail..  Is there some other place this needs to be set, or am I missing something? TIA...

**Comment by MACCAMENZIES** [comment]
I presume you have the startup gcode in Orca?
START_PRINT BED_TEMP=[bed_temperature_initial_layer_single] EXTRUDER_TEMP=[nozzle_temperature_initial_layer]

**Reply by Giterdun2** [reply]
Yes.. exactly that..

**Comment by Tiwatz** [comment]
Can you post the beginning of a g-code file that you sliced in Orca?

**Reply by Giterdun2** [reply]
Sry Tiwatz.. I'm fairly new to KlipperMod.. and Orca as welI.. I would, but I'm not sure how to view the g-code that is generated.. I looked in Mainsail, but not finding it yet.. I have this in Orca... 
BED_MESH_PROFILE LOAD=auto
START_PRINT 
BED_TEMP=[bed_temperature_initial_layer_single]
EXTRUDER_TEMP=[nozzle_temperature_initial_layer]

**Reply by Giterdun2** [reply]
K.. Think I have it.. 

; external perimeters extrusion width = 0.42mm
; perimeters extrusion width = 0.45mm
; infill extrusion width = 0.45mm
; solid infill extrusion width = 0.42mm
; top infill extrusion width = 0.42mm
; first layer extrusion width = 0.50mm

; EXECUTABLE_BLOCK_START
M73 P0 R9
M106 S0
M106 P2 S0
;TYPE:Custom
BED_MESH_PROFILE LOAD=auto
START_PRINT 
BED_TEMP=60
EXTRUDER_TEMP=220
G90
G21
M83 ; use relative distances for extrusion
; filament start gcode
;right_extruder_material: PLA
SET_PRESSURE_ADVANCE ADVANCE=0.025; Override pressure advance value
M106 S0
M106 P2 S0
;LAYER_CHANGE
;Z:0.2
;HEIGHT:0.2
;BEFORE_LAYER_CHANGE
;0.2

**Reply by Tiwatz** [reply]
I'm pretty sure `START_PRINT BED_TEMP=[bed_temperature_initial_layer_single] EXTRUDER_TEMP=[nozzle_temperature_initial_layer]` needs to be on the same line and not on separate lines.

**Reply by Giterdun2** [reply]
ok thank you, I will try that and report back.. I put it on it's own line like all the others as I never realized it mattered..  On another subject, is there a step by step on how to set my wifi to static address? I've read what I could find here, but unable to do the ssh connection from windows 10 cmd prompt..

**Reply by Tiwatz** [reply]
> is there a step by step on how to set my wifi to static address? I've read what I could find here, but unable to do the ssh connection from windows 10 cmd prompt..

Not a step by step that I know of.. I recommend you set the static ip in your router instead of the printer. Should be multiple guides if you Google your router model.
I've never done a ssh connection using Windows cmd, only used Putty and never had a problem.

**Reply by THEGxHOxST** [reply]
Try solarputty for windows.

cd /var/lib/iwd
ls
nano "yourwifi".psk
add to the bottom of that file.
[IPv4]
Address=The IP you'd like your printer to have
Netmask=255.255.255.0
Gateway=your router's IP
Broadcast=your gateways subnet X.X.X.255
DNS=1.1.1.1

**Comment by gpena208777** [comment]
Same issue. Same startup Gcode as well.

**Reply by THEGxHOxST** [reply]
START_PRINT BED_TEMP={first_layer_bed_temperature[0]} EXTRUDER_TEMP={first_layer_temperature[0]}

**Comment by Giterdun2** [comment]
> Try solarputty for windows.
> 
> cd /var/lib/iwd ls nano "yourwifi".psk add to the bottom of that file. [IPv4] Address=The IP you'd like your printer to have Netmask=255.255.255.0 Gateway=your router's IP Broadcast=your gateways subnet X.X.X.255 DNS=1.1.1.1

great.. thank you.. I'll give it a try..

**Comment by Giterdun2** [comment]
> > is there a step by step on how to set my wifi to static address? I've read what I could find here, but unable to do the ssh connection from windows 10 cmd prompt..
> 
> Not a step by step that I know of.. I recommend you set the static ip in your router instead of the printer. Should be multiple guides if you Google your router model. I've never done a ssh connection using Windows cmd, only used Putty and never had a problem.
Ok thanks Tiwatz.. I can do it in the router, but would just prefer to just do it on the printer.. too much other stuff that I don't wanna mess up.. lol..  I'll try THEGxHOxST's suggestion.. If that fails, I'll mess with the router.. ;)

**Comment by Giterdun2** [comment]
> Try solarputty for windows.
> 
> cd /var/lib/iwd ls nano "yourwifi".psk add to the bottom of that file. [IPv4] Address=The IP you'd like your printer to have Netmask=255.255.255.0 Gateway=your router's IP Broadcast=your gateways subnet X.X.X.255 DNS=1.1.1.1

Thank you.. worked like a charm.. ;)

**Reply by THEGxHOxST** [reply]
Glad to hear it!

**Comment by Giterdun2** [comment]
> I'm pretty sure `START_PRINT BED_TEMP=[bed_temperature_initial_layer_single] EXTRUDER_TEMP=[nozzle_temperature_initial_layer]` needs to be on the same line and not on separate lines.

k.. I put them on the same line, still did the same thing..  only heated up to 200.. 
BED_TEMP=[bed_temperature_initial_layer_single]EXTRUDER_TEMP=[nozzle_temperature_initial_layer]
 
Not sure if it's related, but I know that 200 is a default temp in the macros.cfg file.. 
{% set extruder_temp = params.EXTRUDER_TEMP|default(200)|float %}  ; extruder temp, usually set by slicer

**Reply by THEGxHOxST** [reply, marked as answer]
Looks like the slicer isn't send the right temps, causing the start macro to default.

Try this start Gcode in orca instead.

START_PRINT BED_TEMP={first_layer_bed_temperature[0]} EXTRUDER_TEMP={first_layer_temperature[0]}

**Reply by Giterdun2** [reply]
Bingo.. That worked.. Thank you very much.. much appreciated..

**Reply by Giterdun2** [reply]
Sry for the 2 different topics in the post.. Just signed up so a bit new here.. Will keep separate from now on.. Thanks very much for the solutions for both issues..

**Reply by THEGxHOxST** [reply]
Hey, no worries! as long as I could help I'm happy to.
